packages/bible-book-names-intl/src/data/translations/de.json

```diff
@@ -0,0 +1,548 @@
+{
+  "language": "de",
+  "1": {
+    "name": "Mose",
+    "shortNames": [
+      "Ge",
+      "Gen",
+      "Mo"
+    ],
+    "startNumber": 1
+  },
+  "2": {
+    "name": "Mose",
+    "shortNames": [
+      "Ex",
+      "Exo",
+      "Mo"
+    ],
+    "startNumber": 2
+  },
+  "3": {
+    "name": "Mose",
+    "shortNames": [
+      "Le",
+      "Lev",
+      "Mo"
+    ],
+    "startNumber": 3
+  },
+  "4": {
+    "name": "Mose",
+    "shortNames": [
+      "Nu",
+      "Num",
+      "Mo"
+    ],
```

Comment:
I see, the duplication is from the first 5 books from Mose,

Reply:
I am not sure in German, does it have a book name like `Genesis` for `1 Mose`

Reply:
Yes, `Genisis` is also possible, but less common. `1. Mose` would be the preferred book name. 

If  `startNumber` is not working for the 5 books from Mose, we could call it `Genesis` in the `name` field and `1. Mose` in `shortNames`.

Reply:
> I am not sure in German, does it have a book name like `Genesis` for `1 Mose`

The books from Moses are also called Genesis, Exodus, ... 
But, like stated by Tecur, it is not common to reference them in this way (at least in evangelical churches).
Genesis1:1 is written 1.Mose1,1
A Paragraph: 2.Mose5,1-4

Reply:
@Tecur 

> Yes, `Genisis` is also possible, but less common. `1. Mose` would be the preferred book name.
> 
> If `startNumber` is not working for the 5 books from Mose, we could call it `Genesis` in the `name` field and `1. Mose` in `shortNames`.



Do you mind move it it to shortNames? :)

Reply:
Sure. What is the correct syntax in this case? `"1. Mose"`, `"1.Mose"` or `"1Mose"`?

Reply:
@blankton2 Could you give me write access to your fork so I can add these changes directly to this PR?

I would also add some more abbreviations and name variants (according to the [german Wikipedia article](https://de.wikipedia.org/wiki/Liste_biblischer_B%C3%BCcher))
e.g. Zefanja <-> Zephanja